.github/workflows/update-seqr-helm-version.yml

```diff
@@ -0,0 +1,39 @@
+name: Update seqr Version
+on:
+  push:
+    branches:
+      - seqr-helm-version
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: get seqr version
+        run: |
+          seqr_version=$(grep "SEQR_VERSION[ ]*=[ ]*'.*'" ./settings.py | sed -n "s/^.*'\(.*\)'/\1/p")
+          commit_id=$(git rev-parse --short HEAD)
+          echo "SEQR_VERSION=$seqr_version-$commit_id" >> $GITHUB_ENV
```

Comment:
This is cool, I didn't know you could add items to the runner's environment variables like this!